aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -690,6 +690,21 @@ define([],function() {
       SELECTED_DOCUMENTS_CHANGED: "ALF_SELECTED_FILES_CHANGED",
 
       /**
+       * Use by the [AlfGalleryViewSlider]{@link module:alfresco/documentlibrary/AlfGalleryViewSlider}
+       * to publish changes in the number of columns that should be shown in the 
+       * [AlfGalleryView]{@link module:alfresco/documentlibrary/views/AlfGalleryView}.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.40
+       *
+       * @event
+       * @property {number} value - The number of columns to display.
```

Comment:
The hyphen isn't necessary here (or in the many other property tags in this file with hyphens), however I've checked the generated documentation and it doesn't seem to affect it. I would suggest though we try to be consistent, and we don't use hyphens anywhere else (or at least don't need to, and normally don't), so would suggest we just quickly strip out the hyphens in this file at some point. I don't think it should stop this PR being merged though.